docs/lang/articles/contribution/dev_install.md

```diff
@@ -3,74 +3,95 @@ sidebar_position: 3
-
-  ```bash
-  python3 -m pip install --user setuptools astor pybind11 pylint sourceinspect
-  python3 -m pip install --user pytest pytest-rerunfailures pytest-xdist yapf
-  python3 -m pip install --user numpy GitPython coverage colorama autograd
-  ```
-
+  - Install Python dependencies for developers:
 
+    ```bash
+    python3 -m pip install --user -r requirements_dev.txt
+    ```
 
-2.  Make sure you have `clang` with version \>= 7:
+2.  Make sure you have `clang` with version \>= 7 on Linux or download clang-10 on Windows:
 
   - On Ubuntu, execute `sudo apt install libtinfo-dev clang-8`.
   - On Arch Linux, execute `sudo pacman -S clang`. (This is
     `clang-10`).
   - On other Linux distributions, please search [this
     site](https://pkgs.org) for clang version \>= 7.
+  - On Windows please download [clang-10](https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/clang-10.0.0-win.zip). Make sure you add the `bin` folder containing `clang.exe` to the `PATH` environment variable.
+
 
 :::note
-Note that on Linux, `clang` is the **only** supported compiler for
-compiling the Taichi compiler.
+Note that on Linux, `clang` is the **only** supported compiler for compiling the Taichi compiler.
 :::
 
+
 3. Make sure you have LLVM 10.0.0. Note that Taichi uses a **customized
   LLVM** so the pre-built binaries from the LLVM official website or
   other sources probably won't work. Here we provide LLVM binaries
   customized for Taichi, which may or may not work depending on your
   system environment:
-  - [LLVM 10.0.0 for
-    Linux](https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-linux.zip)
+  - [LLVM 10.0.0 for Linux](https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-linux.zip)
+  - [LLVM 10.0.0 for macOS](https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-macos.zip)
+  - [LLVM 10.0.0 for Windows MSVC 2019](https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-msvc2019.zip)
+
+  :::note
+  On Windows, if you use the pre-built LLVM for Taichi, please add
+  `$LLVM_FOLDER/bin` to `PATH`. Later, when you build Taichi please
+  use `cmake -DLLVM_DIR=$LLVM_FOLDER/lib/cmake/llvm`.
+  :::
 
 - If the downloaded LLVM does not work, please build from source:
+  - For Linux:
 
     ```bash
+    # For Linux & OSX
     wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/llvm-10.0.0.src.tar.xz
     tar xvJf llvm-10.0.0.src.tar.xz
     cd llvm-10.0.0.src
     mkdir build
     cd build
-    cmake .. -DLLVM_ENABLE_RTTI:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_TERMINFO=OFF
+    cmake .. -DLLVM_ENABLE_RTTI:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE=Release   -DLLVM_TARGETS_TO_BUILD="X86NVPTX" -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_TERMINFO=OFF
```

Comment:
It is `-DLLVM_TARGETS_TO_BUILD="X86;NVPTX"` :-)

Reply:
Nice catch! Thanks!